### DISCUSSION TITLE
官方文档有一些错别字

### DISCUSSION THREAD

**Discussion by xmsz** [go-kratos/kratos#1358] [General]
https://go-kratos.dev/docs/guide/ent
![image](https://user-images.githubusercontent.com/5879146/129649980-b0c0106c-7459-4553-9d22-95b40e0e8d64.png)

**Comment by xmsz** [comment]
还有
![image](https://user-images.githubusercontent.com/5879146/130029294-675824b4-670a-4f7f-9d7a-eb310d0c36b5.png)

gprc

**Comment by Windfarer** [comment]
已修复

**Reply by xmsz** [reply]
点赞！
